lib/js-api/src/api/gs-core-rest-api/docs/Hold.md

```diff
@@ -4,19 +4,19 @@
 
 ```ts
 export interface Hold {
```

Comment:
You can remove the whole typescript block, putting the source of the interface in the docs does not make sense

Reply:
done